engine-tests/src/tests/erc20.rs

```diff
@@ -12,8 +12,9 @@ use aurora_engine_types::account_id::AccountId;
 use aurora_engine_types::parameters::connector::{
     Erc20Identifier, Erc20Metadata, SetErc20MetadataArgs,
 };
-use aurora_engine_types::parameters::engine::SetOwnerArgs;
+use aurora_engine_types::parameters::engine::{SetOwnerArgs, TransactionStatusEvmErrorKind};
```

Comment:
Could `TransactionStatusEvmErrorKind` be just `EvmErrorKind` cause it looks wired because the `TransactionStatusEvmErrorKind` is used in `TransactionStatus`?